addons/point_of_sale/tests/common.py

```diff
@@ -285,11 +285,57 @@ def _create_taxes(self):
 
         tax7: 7%, excluded in product price
         tax10: 10%, included in product price
+        tax21: 21%, included in product price
         """
-        tax7 = self.env['account.tax'].create({'name': 'Tax 7%', 'amount': 7})
-        tax10 = self.env['account.tax'].create({'name': 'Tax 10%', 'amount': 10, 'price_include': True, 'include_base_amount': False})
-        (tax7 | tax10).mapped('invoice_repartition_line_ids').write({'account_id': self.tax_received_account.id})
-        (tax7 | tax10).mapped('refund_repartition_line_ids').write({'account_id': self.tax_received_account.id})
+        def create_tag(name):
+            return self.env['account.account.tag'].create({
```

Comment:
Since the creation of tags is only during testing, maybe you can `sudo` it to avoid the access rights error that fails the test.

Reply:
You are right, there a problem with access right but it's due to the user being change to the PoS manager too early while the configuration isn't done yet.

Instead of using sudo, I have delayed the moment we switch to the PoS manager user, it should solves the issue.
Thanks for insight btw.